pkg/cmd/cmd_push.go

```diff
@@ -127,14 +113,47 @@ func pushTarPackage(ociUrl, localTarPath string, settings *settings.Settings) er
 		return err
 	}
 
-	// 2. Generate the OCI options from oci url and the version of current kcl package.
+	// 2. push the package
+	return pushPackage(ociUrl, kclPkg, settings)
+}
+
+// pushPackage will push the kcl package to the oci registry.
+func pushPackage(ociUrl string, kclPkg *pkg.KclPkg, settings *settings.Settings) error {
+	// 1. Package the current kcl package into default tar path.
```

Comment:
U can put documents to the function documents, just like:
// pushPackage will push ..
// 1.
// 2.
// 3.
// 4.

Reply:
fixed.